jenkins/scripts/llnode-continuous-integration.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash -ex
+
+rm -rf llnode
+
+"$(dirname $0)"/common/getNode.sh # Download node tarball.
+
+export PATH="$(cd $WORKSPACE/node-*/bin; pwd):$PATH"
+
+export NPM_CONFIG_USERCONFIG=$WORKSPACE/npmrc
+export NPM_CONFIG_CACHE=$WORKSPACE/npm-cache
+node -v
+npm -v
+export npm_loglevel=error
+npm set progress=false
+ls
+
+git clone --depth=1 -b $GIT_BRANCH https://github.com/$GIT_REPO.git llnode
+cd llnode
+
+[ $(uname) = Darwin ] && git clone --depth=1 -b release_39 https://github.com/llvm-mirror/lldb.git lldb
```

Comment:
I think we don't actually need the tools cloning, configuring and build steps now.  `npm install` would do all of that. As long as the machine has a lldb installed, `scripts/configure.js` in llnode would grab the right version of headers, `gyp_llnode` and `make` would be run during `npm install` as well.

Reply:
As for the tests, I think on Linux we would need to set `TEST_LLDB_BINARY` to `which lldb-3.9` before running `npm test`. Also it would be helpful to set `TEST_LLNODE_DEBUG=true` and `LLNODE_DEBUG=true` if printing stuff to stderr does not interfere with the job.